### ISSUE TITLE
resnet training encounter error 

### ISSUE THREAD

**Issue by andyhahaha** [xuyuwei/resnet-tf#2]
![image](https://cloud.githubusercontent.com/assets/14272969/19145338/17f3e1fe-8be1-11e6-8afc-0473c725f44e.png)

There is a data dimension mismatch.

**Comment by andyhahaha** [comment]
I found that it is because the training data size can't be divided by batch_size=128.
So that,  it can't find the last training batch 49920 ~ 50048.
To solve this problem, I zero padding those 50000 ~ 50048 data, and it can train nicely.

This how I pad the train data and test data

`X_train = np.pad(X_train,pad_width=[[0,48],[0,0],[0,0],[0,0]], mode='constant', constant_values=0)
    Y_train = np.pad(Y_train,pad_width=[[0,48],[0,0]], mode='constant', constant_values=0)
    X_test = np.pad(X_test,pad_width=[[0,16],[0,0],[0,0],[0,0]], mode='constant', constant_values=0)
    Y_test = np.pad(Y_test,pad_width=[[0,16],[0,0]], mode='constant', constant_values=0)
`
